script/Ciel/Day 5/05_01_CIEL05_4D.txt

```diff
@@ -643,16 +643,15 @@ So that's your angle.
 [sha:aeef30a9d8752aac2b8e6a0a2b02338b6f9082c2]{
 -- Page 832, Offset 13092.
 -- オレはお忍び。テメェらは私的な暴走。<ヤ|・><ツ|・>を追っている事を上に知られるのは都合が悪いってコトだろう？」
-// Interpreting お忍び as 'I will act as though I am inside the palanquin and therefore can't see any of this shit'
-// over 'I will actively try and bundle this shit into the palanquin'
-I'm just gonna look the other way on this one.
+// Kotowaza here is travelling incognito
+What I'm up to here ain't exactly above board either, I'm not about to kick up a fuss.
  Whatever sketchy shit you're up to is your own business.
  It'd be a real problem if the higher-ups caught wind of the fact that you're going after %{i}him%{/i}, right?"
```

Comment:
This is a bit overedited.
「マヌケ、今ので同じ穴のムジナだって分かれよ。
   オレはお忍び。テメェらは私的な暴走。<ヤ|・><ツ|・>を追っている事を上に知られるのは都合が悪いってコトだろう？」
Let's get into the previous line as well, since the way it connects to this one is crucial.

"What, you retarded? I'm saying we're stuck in the same shit―――
I'm here undercover, while you guys are out stirring shit up 'cause of your own interests.
It'd be a real problem if the higher-ups caught wind of the fact that you're going after %{i}him%{/i}, right?"